src/lib/adapters/WebDav.js

```diff
@@ -0,0 +1,400 @@
+/* @jsx el */
+
+import InputInitializeHook from '../InputInitializeHook'
+import Bookmark from '../Bookmark'
+import humanizeDuration from 'humanize-duration'
+
+const {h} = require('virtual-dom')
+
+function el (el, props, ...children) {
+    return h(el, props, children)
+};
+
+const url = require('url')
+
+function getServerKey (server) {
+    let key = "";
+
+    key = key + "," + server.url;
+    key = key + "," + server.username;
+    key = key + "," + server.bookmark_file;
+
+    return key;
+}
+
+function getBookmarkKey (bm) {
+    let key = "";
+
+    key = bm.path;
+    key = "," + bm.title;
+    key = "," + bm.url;
+
+    return key;
+}
+
+var WebDavAdapters = new Map ();
+
+export default class WebDavAdapter {
+    constructor (server) {
+        console.log('Webdav constructor');
+        console.log(server);
+
+        this.server = server;
+        this.db = new Map();
+
+        console.log ("THIS");
+        console.log (this);
+    }
+
+    // WebDavAdapter is a multiton based in server
+    static getMultiton (server) {
```

Comment:
This is no longer necessary, as the Account itself is now a multiton, cached by account id.

Reply:
I will remove the multiton code.